ceno_zkvm/src/bin/e2e.rs

```diff
@@ -40,6 +42,11 @@ struct Args {
     #[arg(short, long)]
     max_steps: Option<usize>,
 
+    // Profiling granularity.
+    // Setting any value restricts logs to profiling information
```

Comment:
I think you might want to control log verbosity and profiling information independently?

(It's reasonable to default log verbosity to silent, if you turn on profiling, though.)

Reply:
Ideally you would, but the only way to keep the profiling data readable is to filter out some of the spans. And filtering out some of the spans has an unclear impact (to me at least) on the general logging context. I think it's simpler and not that inconvenient to keep them separate.